activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   AssociatedValidator accepts `forward_validation_context` flag
```

Comment:
Would you mind updating the CHANGELOG to include some more information?
I think what you have [here](https://github.com/rails/rails/pull/24135/files#diff-505967108be6686c7d9744aa378f0575R65-R66) is very clear on what this feature does.